packages/snowpack/tsconfig.json

```diff
@@ -6,8 +6,6 @@
     "esModuleInterop": true,
     "strict": true,
     "noImplicitAny": false,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
```

Comment:
Why were these removed?

Reply:
not meant to be committed, but did want to run this idea by you in a separate PR, will follow up